code/modules/hydroponics/grown/citrus.dm

```diff
@@ -117,9 +117,6 @@
 /obj/item/reagent_containers/food/snacks/grown/firelemon/attack_self(mob/living/user)
 	user.visible_message("<span class='warning'>[user] primes [src]!</span>", "<span class='userdanger'>You prime [src]!</span>")
 	log_bomber(user, "primed a", src, "for detonation")
-	if(iscarbon(user))
-		var/mob/living/carbon/C = user
-		C.throw_mode_on()
```

Comment:
Why are you removing this? When you prime a grenade, throw mode should be entered.

Reply:
Uhh, no this is specific to this fruit for whatever reason. Regular grenades don't toggle your throw mode, so it's for consistency's sake.

Reply:
Regular grenades can be thrown while primed, even if not in throw mode.